src/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionEntity.tsx

```diff
@@ -37,7 +37,14 @@ export class ArtistCollectionEntity extends React.Component<CollectionProps> {
       title,
       artworksConnection,
     } = this.props.collection
-    const artworks = artworksConnection.edges.map(({ node }) => node)
+    const artworks = get(artworksConnection, ac =>
```

Comment:
Added this to fix the crash loop we're seeing due to the current elasticsearch issues on staging (would be an error on prod too, so good to fix!)